frontend/packages/core/src/button.tsx

```diff
@@ -115,6 +115,20 @@ const StyledBorderButton = styled(StyledButton)({
 
 /** Provides feedback to the user in regards to the action of the button. */
 type ButtonVariant = "neutral" | "primary" | "danger" | "destructive" | "secondary";
+type IconButtonSize = "small" | "medium";
```

Comment:
nit: 
```suggestion
type IconButtonSize = keyof typeof ICON_BUTTON_STYLE_MAP;
```

Reply:
Ooh that's cool, did not know about the `keyof` operator